### ISSUE TITLE
does it compatiblw with sc6138a?

### ISSUE THREAD

**Issue by djtrance** [vatanuki/silan#2]
I have this:
[audio rush 5.1 decoder](https://es.aliexpress.com/item/32828039888.html?spm=a2g0o.productlist.0.0.7bdf5784nxBmDU&algo_pvid=e11754b1-4486-4793-8a24-b0b0c250d42a&algo_expid=e11754b1-4486-4793-8a24-b0b0c250d42a-0&btsid=0ab6f82415948576904828062e4efe&ws_ab_test=searchweb0_0,searchweb201602_,searchweb201603_)

and I what to add some functions, but when i use the serial port I only see something like this: `Clocks: CPU:150MHz, SDR:150MHz, DSP:150MHz, PLL:300MHz
Bootloader version: nbap_0.4.0_sc6138a-spdif-000_20151024.1508

boot up with partition 1

dstaddr:0xa1c00000, srcadg⸮⸮⸮⸮⸮0⸮L⸮⸮. nenth;0x⸮⸮u⸮Lc⸮⸮⸮⸮⸮su}:0x60⸮3vs87#memv⸮⸮⸮⸮y⸮տ{{⸮Z⸮?⸮leck⸮w⸮0⸮⸮030fk8v!ok-⸮
N[nbu\WiIMIIi]YIZVRrZ*+⸮%%isRR⸮⸮{⸮[n⸮ap] * Hello World!
[nbap] *******************.+***⸮*⸮
_⸮}]R⸮⸮⸮V+⸮H(⸮kх⸮⸮⸮irq hcndlgr32<⸮⸮⸮714⸮}0⸮sk⸮⸮⸮⸮⸮⸮⸮p~oze gone
[nbap]silan_dsp_probe -> 0
Init Codec
Dea⸮{lmonu~//"⸮⸮⸮onvig;301,-1
DR+K⸮⸮⸮⸮⸮⸮2,50⸮⸮
slmno⸮s⸮oa}彥⸮⸮one0⸮⸮Codec ready
[nbap]sil}~_s˝⸮}⸮⸮⸮⸮⸮-⸮⸮0
⸮Li⸮⸮⸮⸮⸮vg⸮⸮⸮[⸮b⸮⸮⸮⸮⸮⸮}⸮⸮.0⸮.7r⸮1generao_24⸮2⸮>⸮/⸮4⸮⸮⸮⸮/⸮}ws}٥ɵ⸮⸮ɕ}⸮⸮⸮⸮}⸮⸮⸮⸮⸮ <
Stast0DSP/..
DSP online
[lib]send cmd = 17409
silan_dsp_ioctl cmd 17409,   0====
dsp_send_cmd 1
mbox_tx_work start 
[DSP]=======recv irq =========
[DSP] dsp_mbox_recv cmd 1 
[DSP] decode_start 
cadsp_recv_cmd 1
dsp_recv_cmd 1 end
ll cmd break
DSP version: dspfw_00.00.0501_general_20151102.1551
dsp_send_cmd 1 end, ret 0
`

And did find the sdk or other information about it

**Comment by csorvasig** [comment]
Hello,

Did you find any documentation for this chip? I need the datasheet at first.

Thanks,
Gabor

**Comment by djtrance** [comment]
sorry I couldnt find it.. the only thing I did, was to check if in the serial port I could have access to the linux system but I have not luck on this.

**Comment by csorvasig** [comment]
ok, thank you.
My problem is with i2s audio output signals. I put a DIT (CS8406 pr WM8805) to the i2s outputs, but the spdif stream is cracking... the DIT can not synchronise to the i2s clocks. (MCLK, LRCK, BCK). I checked the signal timing with scope, and it looks good, but I think the SDATA or the sync clocks are not stable or not standard.

**Comment by djtrance** [comment]
did you set up de Dac with the i2c bus before to send it the data to i2s port?
could you show me your board? did you have access to linux system?

**Comment by csorvasig** [comment]
CS8406 has software or hardware control mode, I use it in hardware control mode. Of course I set up all parameters.
![IMG_5930](https://user-images.githubusercontent.com/62664393/154466854-292373a8-adee-4a05-ab4c-a1bf29562951.jpg)

WM8805 has i2s (DSP) input directly to spdif output, without any setting possibility. 
![IMG_5929 (1)](https://user-images.githubusercontent.com/62664393/154467679-3ceab932-1a6e-4b0d-bb24-5335bfcf0bbd.jpg)

I tried both DIT circuit, but without success.

I do not have access to linux system. 
Here you are my decoder board.
![IMG_5928](https://user-images.githubusercontent.com/62664393/154463988-3ba1151b-9a87-4336-ae85-ed660cba9e75.jpg)
I take the i2s signals from the DAC legs.  
The analog output from the DAC is good, no cracking noise.

**Comment by djtrance** [comment]
did you try to connect it before the 􏰛􏰚􏰞􏰖􏰕􏰖􏰑􏰒pe65612? 
did you check it the CS8406 with other source?

**Comment by csorvasig** [comment]
Yes, this CS8406 board is working in a Denon DN-A7100 sound processor for a 10 years… so it is working good.

It is very strange for me, that the own DA converter on the SC6136A board is working without noise.
I test the AES signal with NTI DL-1 Digilyzer instrument and it can not catch the sync, just sometimes.

when i start this SC6136A board it has a built in 2-3sec sound effect at the end of the starting process. This sound has 22kHz sample rate and it sounds good on Digilyzer. It can catch the sync. But after the starting process the board change source to HDMI (DVD player) and the 48k sync falls apart on Digilyzer. The DAC works good, just the DIT can not sync to the DSP’s i2s bus.

i do not understand the reason of this because on oscilloscope i see good all the i2s syncs (MCLK, LRCK, BCK) and data line.
I can recognise the i2s timing.
